lib/src/components/Table/Table.css.ts

```diff
@@ -0,0 +1,48 @@
+import { globalStyle, style, createTheme } from "@vanilla-extract/css";
+import { ITableContainerThemeVars, ITableThemeVars } from "./Table.types";
+
+export const [, tableContainerThemeVars]: ITableContainerThemeVars =
+  createTheme({
+    customBorderRadius: "4px",
+    customBorderColor: "black",
+  });
+
+export const [, tableThemeVars]: ITableThemeVars = createTheme({
```

Comment:
descructure the class name and add it to the container. Helps scope the variables